pegasus/sites.v3/code.org/views/tabs_section/resources/resources_tabs.haml

```diff
@@ -1,9 +1,10 @@
 -# Can only be used on rebranded pages that use the
 -# "design-system-pegasus.scss" stylesheet.
 
-%h2=hoc_s(:resources_tabs_heading)
+%h2
+  #{resources_heading}
 %p.desc
-  =hoc_s(:resources_tabs_desc, markdown: :inline, locals: {sign_up_url: CDO.studio_url("/users/sign_up")})
+  #{resources_desc}
```

Comment:
Could each file that uses this view just set the `resources_tabs_heading` and `resources_tabs_desc` variables like `= view :"tabs_section/resources/resources_tabs", resources_tabs_heading: [value], resources_tabs_desc: [value]`? I think I'm mainly wondering why `resources_heading: hoc_s(:resources_tabs_heading), resources_desc: hoc_s(:resources_tabs_desc, markdown: :inline, locals: {sign_up_url: CDO.studio_url("/users/sign_up")})` needs to be pulled out of this file and passed in from each file that uses the `resources_tabs` view